mmcv/image/geometric.py

```diff
@@ -38,14 +38,24 @@ def _scale_size(size, scale):
 }
 
 if Image is not None:
-    pillow_interp_codes = {
-        'nearest': Image.NEAREST,
-        'bilinear': Image.BILINEAR,
-        'bicubic': Image.BICUBIC,
-        'box': Image.BOX,
-        'lanczos': Image.LANCZOS,
-        'hamming': Image.HAMMING
-    }
+    if hasattr(Image, 'Resampling'):
```

Comment:
We also need to add some comments to recode why we need to check whether `Image` has this attribute `Resampling` like the explanation in the PR message.

Reply:
@zhouzaida Good point, I have now added comments to geometric.py